src/expr/CMakeLists.txt

```diff
@@ -168,6 +168,16 @@ add_custom_command(
   DEPENDS mkmetakind metakind_template.cpp metakind.h ${KINDS_FILES}
 )
 
+add_custom_command(
+  OUTPUT node_manager.cpp
+  COMMAND
+    ${mkmetakind_script}
+    ${CMAKE_CURRENT_LIST_DIR}/node_manager_template.cpp
+    ${KINDS_FILES}
+    > ${CMAKE_CURRENT_BINARY_DIR}/node_manager.cpp
```

Comment:
to my knowledge, using such shell built-ins is discouraged in cmake commands. Instead, the script should directly write to that file...

Reply:
Hmm, this just copies what we do for the other commands. How about I clean that up in a future PR?